src/core/renderers/DroppableLabel.tsx

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Wouldn't it make sense to adapt the tester of the https://github.com/eclipsesource/jsonforms-editor/blob/master/src/core/renderers/DroppableControl.tsx to also handle the Label. The two classes are identical otherwise.
Maybe make the DroppableControl the default for all uischema elements, so that we support all kind of renderers out of the box without the need to wrap everyone of them?

Reply:
Sounds good to me. I already used that approach for the `DroppableArrayControl` which I registered with a lower prio than `DroppableControl` to reuse its code. I guess we could rename `DroppableControl` to `DroppableElement` and do that for all elements.